README.md

```diff
@@ -100,3 +100,40 @@ a template to create your ``.env`` file. For Bootcamp to work, it needs 6 values
 #### 8) Go to the home page of your bootcamp site and login via EdX
   - Log in as staff@example.com (password: edx).
   - You should see a message that you owe the amount of money equal to the installment you created in step 6.
+
+
+## Integration with FluidReview
+
+#### 1) Set FluidReview environment variables
+Values for the first four can be found at `../admin/developer/` on the FluidReview site.
+  - `FLUIDREVIEW_CLIENT_ID`
+  - `FLUIDREVIEW_CLIENT_SECRET`
+  - `FLUIDREVIEW_ACCESS_TOKEN`
+  - `FLUIDREVIEW_REFRESH_TOKEN`
+  - `FLUIDREVIEW_BASE_URL` (the FluidReview site URL)
+  - `FLUIDREVIEW_WEBHOOK_AUTH_TOKEN` (a manually assigned value that is difficult to guess)
+  
+#### 2) Create a trigger in FluidReview
+  - Go to `../admin/triggers/`
```

Comment:
Similar to line 108, `<BOOTCAMP_ADMISSION_BASE_URL>/admin/triggers/`

Reply:
✅ , it should actually be `<FLUIDREVIEW_BASE_URL>` here and above.